python/sglang/srt/managers/tp_worker_overlap_thread.py

```diff
@@ -62,7 +62,7 @@ def __init__(
         # Load the model
         self.worker = TpModelWorker(server_args, gpu_id, tp_rank, dp_rank, nccl_port)
         self.max_running_requests = self.worker.max_running_requests
-        self.device = self.worker.device
+        self.device = self.worker.device if self.worker.device != "hpu" else "cpu"
```

Comment:
Question: dose hpu platform use cpu as device_type for dispatching?

Reply:
The same as the below explanation. I'll try to make it clear.